model_compression_toolkit/gptq/keras/quantization_facade.py

```diff
@@ -94,19 +96,37 @@ def get_keras_gptq_config(n_epochs: int,
             >>> import model_compression_toolkit as mct
             >>> import tensorflow as tf
 
-            Create a GradientPTQConfigV2 to run for 5 epochs:
+            Create a GradientPTQConfig to run for 5 epochs:
 
             >>> gptq_conf = mct.gptq.get_keras_gptq_config(n_epochs=5)
 
             Other Tensorflow optimizers can be passed:
 
             >>> gptq_conf = mct.gptq.get_keras_gptq_config(n_epochs=3, optimizer=tf.keras.optimizers.Nadam())
 
-            The configuration can be passed to :func:`~model_compression_toolkit.keras_post_training_quantization` in order to quantize a keras model using gptq.
+            To enable Gradual Activation Quantization with non-default settings build GradualActivationQuantizationConfig:
+
+            >>> gradual_act_conf = mct.gptq.GradualActivationQuantizationConfig(mct.gptq.QFractionLinearAnnealingConfig(initial_q_fraction=0.2))
+            >>> gptq_conf = mct.gptq.get_keras_gptq_config(n_epochs=3, gradual_activation_quantization=gradual_act_conf)
+
+            The configuration can be passed to :func:`~model_compression_toolkit.keras_gradient_post_training_quantization` in order to quantize a keras model using gptq.
+
 
         """
+        optimizer = optimizer or tf.keras.optimizers.Adam(learning_rate=LR_DEFAULT)
```

Comment:
If we are touching this API, lets make sure that this is aligned with the arguments and values of the parallel pytorch API (it might be fine but just lets be sure)

Reply:
I have no idea how the values were chosen and why they differ between the frameworks. Let's align both defaults to the default value in the accuracy runner (I think its 1e-2 for both lr and reg factor but double check that)